docs/src/features/atlases.md

```diff
@@ -11,9 +11,9 @@ For an atlas ``\mathcal A`` we further require that
 ```
 
 We say that ``φ_i`` is a chart about ``p``, if ``p\in U_i``.
-An atlas provides a connection between a manifold and the Euclidean space ``\mathbb{R}^n``, since
+An atlas provides a connection between a manifold and the Euclidean space ``ℝ^n``, since
 locally, a chart about ``p`` can be used to identify its neighborhood (as long as you stay in ``U_i``) with a subset of a Euclidean space.
-Most manifolds we consider are smooth, i.e. any change of charts ``φ_i \circ φ_j^{-1}: \mathbb{R}^n\to\mathbb{R}^n``, where ``i,j\in I``, is a smooth function. These changes of charts are also called transition maps.
+Most manifolds we consider are smooth, i.e. any change of charts ``φ_i \circ φ_j^{-1}: ℝ^n\toℝ^n``, where ``i,j\in I``, is a smooth function. These changes of charts are also called transition maps.
```

Comment:
It might not render correctly without a space after `\to`.

Reply:
That is probably correct, yes. But the `\to` should/could also become its actual error symbol then, Will  search/replace that as well then.